catalog/imaging/imaging-transformer-nitf/pom.xml

```diff
@@ -98,13 +110,16 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
                             thumbnailator,
+                            catalog-core-api,
```

Comment:
We shouldn't have to embed the catalog-core-api bundle.  I think alternatively you can add this maven dependency into the imaging-app as a prereq to fix runtime issues.